### PR TITLE
feat: use pinned GA actions instead of version

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

#### Summary

use pinned GA actions instead of version

#### Ticket Link

Fixes

#### Release Note

```release-note
feat: use pinned GA actions instead of version
```

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -35,10 +35,12 @@ jobs:
         run: "helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx"
 
       - name: Create KIND Cluster
-        uses: helm/kind-action@v1.2.0
+        uses: helm/kind-action@94729529f85113b88f4f819c17ce61382e6d8478 # pin@v1.2.0
 
       - name: Install Ingress Controller
-        run: "helm install ingress-nginx/ingress-nginx --generate-name --set controller.service.type='NodePort' --set controller.admissionWebhooks.enabled=false"
+        run: "helm install ingress-nginx/ingress-nginx --generate-name --set
```
Comment (sabre1041) [inline .github/workflows/test.yml:41]: @developer-guy if this is going to be broker out into multiple lines, can the `--set` align?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Small change requested

File: .github/workflows/test.yml (hunk 1 above)
Comment (developer-guy) [reply .github/workflows/test.yml]: ah yes, sorry about that, it is my formatter bad 🤦‍♂️

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**